includes/class-wc-payment-token-wcpay-link.php

```diff
@@ -83,4 +83,29 @@ public function get_email( $context = 'view' ) {
 	public function set_email( $email ) {
 		$this->set_prop( 'email', $email );
 	}
+
+	/**
+	 * Returns redacted/shortened customer email
+	 *
+	 * @param string $context What the value is for. Valid values are view and edit.
+	 * @return string Redacted/shortened customer email.
+	 */
+	public function get_redacted_email( $context = 'view' ) {
+		return $this->redact_email_address( $this->get_email( $context ) );
+	}
+
+	/**
+	 * Transforms email address into redacted/shortened format like ***xxxx@x.com.
+	 * Using shortened length of four characters will mimic CC last-4 digits of card number.
+	 *
+	 * @param string $email Email address.
+	 * @return string Redacted/shortened email address.
+	 */
+	private function redact_email_address( $email ) {
+		$placeholder             = '***';
+		$shortened_length        = 4;
+		list( $handle, $domain ) = explode( '@', $email );
+		$redacted_handle         = strlen( $handle ) > $shortened_length ? substr( $handle, - $shortened_length ) : $handle;
+		return "$placeholder$redacted_handle@$domain";
```

Comment:
⛏️  I noticed that in other lines there's almost always a newline before the return 👀 
```suggestion
		$redacted_handle         = strlen( $handle ) > $shortened_length ? substr( $handle, - $shortened_length ) : $handle;

		return "$placeholder$redacted_handle@$domain";
```